content/en/platform/corda/4.12/enterprise/node/operating/tvu/tvu-behavior.md

```diff
@@ -0,0 +1,35 @@
+---
+date: '2023-12-15'
+menu:
+  corda-enterprise-4-12:
+    identifier: corda-enterprise-4-12-tvu-behavior
+    parent: corda-enterprise-4-12-tvu
+tags:
+- tvu
+- transaction validator utility
+title: Transaction Validator Utility behavior
+weight: 100
+---
+
+# Transaction Validator Utility behavior
+
+This section describes the steps that the TUV goes through when validating transactions present in a specified database.
+
+1. Upon startup, the utility connects to the specified database.
+2. If it is not able to connect to the database, it exits with an error message.
+3. It loads the following:
+    * If `-l` or `--load-tx-time` CLI option is given, it loads last known processed transaction.
+    * If `-i` CLI option is given, it loads transaction ID(s) for reverification.
```

Comment:
And what is the behaviour if non of the optional command line options are given. @suhas-srivastava

Reply:
If none of the `-l`, `-i` or `--load-tx-time` options are given then all transactions are loaded from the given db.

Reply:
Updated